share/spice/isslocation/isslocation.js

```diff
@@ -0,0 +1,36 @@
+(function(env) {
+    "use strict";
+    env.ddg_spice_isslocation = function(api_result) {
+        console.log("API result is", api_result);
+        if (api_result.message !== "success") {
+            return Spice.failed('isslocation');
+        }
+        var issLatitude = (Math.round(api_result.iss_position.latitude * 100) / 100).toFixed(2);
+        var issLongitude = (Math.round(api_result.iss_position.longitude * 100) / 100).toFixed(2);
+
+        Spice.add({
```

Comment:
Yeah it looks like it's just maps. :) Now that I think about it, it should probably pull the data down automatically if we're using the map model kinda like in soundcloud.